### ISSUE TITLE
mistake in listfile ?

### ISSUE THREAD

**Issue by hoxton22** [bloerwald/wow-listfile#105]
item/texturecomponents/torsouppertexture/cloth_raidpriestmythic_o_01_chest_tu_m.blpcharacter/draenei/male/draeneimale_hd_lod01.skin

**Comment by bloerwald** [comment]
Yeah, those sometimes happen. There likely are multiple. Shitty scripts and stuff.

**Comment by bloerwald** [comment]
I do not know how a safeguard would look.

**Comment by bloerwald** [comment]
negative on `\..*\.`: multiple files have those, like `world/maps/eyeofthestorm2.0/eyeofthestorm2.0_31_29.adt`.

I did another manual pass in 9ff313f.
